test/syscalls/linux/dev.cc

```diff
@@ -62,6 +62,8 @@ TEST(DevTest, LseekDevFull) {
 }
 
 TEST(DevTest, LseekDevKmsg) {
+  // TODO: VFS1 doesn't support /dev/kmsg, Remove skip once deleted VFS1.
```

Comment:
Don't need this comment, this will be removed with `IsRunningWithVFS1` when VFS2 is done. Same below.